env.postgres

```diff
@@ -7,3 +7,5 @@ DB_PORT=5432
 DB_DATABASE=lychee
 DB_USERNAME=postgres
 DB_PASSWORD=password
+CACHE_DRIVER=file
+LOG_VIEWER_CACHE_DRIVER=file
```

Comment:
Pretty sure these don't belong here. These are just for testing and are Docker env, not Lychee.